pom.xml

```diff
@@ -44,6 +44,7 @@
         <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
         <java.version>11</java.version>
         <kiwigrid.version>2.5</kiwigrid.version>
+        <logstash-logback-encoder.version>5.1</logstash-logback-encoder.version>
```

Comment:
Please add the latest version 7.2 for the encoder. Please check in maven repo for the latest and re-test it.